lib/lob/resources/resource_base.rb

```diff
@@ -20,8 +20,8 @@ def find(resource_id)
         submit :get, resource_url(resource_id)
       end
 
-      def create(options = {})
-        submit :post, endpoint_url, options
+      def create(options={}, headers={})
+        submit :post, endpoint_url, options, headers
```

Comment:
@mpiercy827 why is `options` optional? Aren't options just the payload and a payload is always required for create requests.